sqlitecluster/SQLiteNode.cpp

```diff
@@ -254,9 +254,6 @@ void SQLiteNode::_replicate(SQLitePeer* peer, SData command, size_t sqlitePoolIn
                         db.rollback();
                     }
                 }
-
-                // Notify that we've succeeded (it actually also notifies if we were canceled, but that's fine).
-                _localCommitNotifier.notifyThrough(db.getCommitCount());
```

Comment:
Before, if `_handleCommitTransaction()` returned anything other than `SQLITE_OK` (i.e. `SQLITE_BUSY_SNAPSHOT`), this notification would still run. Now, it won't. Is that intended? Seems like it might be a problem?

Reply:
We couldn't get here with any other condition than `SQLITE_OK` (see the condition on the loop above) or `SQLiteSequentialNotifier::RESULT::CANCELED` (per the removed comment) which didn't actually do anything, but wasn't harmful (it would notify the same commit count as the most recent successful commit).